Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -158,22 +252,14 @@ public final class System {
     }
 
     ///
-    /// Start the system.  This will query various BRD services, notably the BlockChainDB, to
-    /// establish the available networks (aka blockchains) and their currencies.  If the
-    /// `networksNeeded` array includes the name of an available network, then a `Network`
-    /// will be created for that network.  (This generates a `SystemEvent` which can be used by
-    /// the App to create a `WalletManager`)
-    ///
-    /// This method can be called repeatedly; however ONLY THE FIRST invocation will create
-    /// Networks for needed networks.  Subsequent calls will simple restart `System` processing.
+    /// Configure the system.  This will query various BRD services, notably the BlockChainDB, to
+    /// establish the available networks (aka blockchains) and their currencies.  For each
+    /// `Network` there will be `SystemEvent` which can be used by the App to create a
+    /// `WalletManager`
     ///
-    /// - Parameter networksNeeded: Then needed networks
+    /// @Note: This should only be called one.
     ///
-    public func start (networksNeeded: [String]) {
-        if !networks.isEmpty {
-            managers.forEach { $0.connect() }
-            return
-        }
+    public func configure () {
 
         #if MAINNET
```

Comment:
if MAINNET and TESTNET have cropped up more and more. This paradigm doesn't map to other languages. Would be ideal if we had this parameterized wherever we are using this pattern.

Reply:
We have CORE-284 coming, from Aaron, that eliminates the compiler flag altogether.  BRChainParams changes to include the differences in BITCOIN_TESTNET

Reply:
We still will be providing if it is mainnet to BDB, right?